dist/openshift/cincinnati.yaml

```diff
@@ -201,13 +201,13 @@ objects:
           targetPort: ${{PE_STATUS_PORT}}
       selector:
         deploymentconfig: cincinnati
-  - apiVersion: policy/v1beta1
+  - apiVersion: policy/v1
     kind: PodDisruptionBudget
     metadata:
       name: cincinnati-pdb
       namespace: cincinnati
     spec:
-      minAvailable: 1
+      minAvailable: 3
```

Comment:
For disconnected environments we do not need replica count as 3. So I do not think we need to change this here.

Reply:
removed